### PR TITLE
use go modules with pinned pkg versions

### PR DESCRIPTION
**Pull request by vtolstov**

### CODE REVIEW

**Comment by vtolstov** [comment]
go.mod contains versions of packages from manifest file in go-ovn

**Comment by vtolstov** [comment]
@hzhou8 can you merge this, and optionally tag release?

**Comment by hzhou8** [comment]
Shall this be rebased? It seems the first 3 commits are already merged through the previous PR?

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module github.com/ebay/libovsdb
+
+go 1.12
```
Comment (hzhou8) [inline go.mod:3]: According to https://github.com/golang/go/wiki/Modules#gomod there is no "go" directive in go.mod. Please correct me if I misunderstood anything. I am new to "go mod". Is this file generated by go command or did you update it manually?

File: vendor/modules.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# github.com/cenkalti/hub v1.0.1-0.20160527103212-11382a9960d3
```
Comment (hzhou8) [inline vendor/modules.txt]: I think we should remove vendor directory from the repo, when using go mod. What do you think?

File: go.mod (hunk 1 above)
Comment (vtolstov) [reply go.mod:3]: this is generated automatic by go mod init github.com/ebay/libovsdb
and started from go 1.12 this string automatic added to go.mod file

File: vendor/modules.txt (hunk 2 above)
Comment (vtolstov) [reply vendor/modules.txt]: yes, thanks i missed this. vendor dir needed only in final product that used this library

**Comment by vtolstov** [comment]
i'm rebase and cleanup

**Pull request merged**